tigerpath/majors_and_certificates/majors/NES_2018.json

```
@@ -0,0 +1,213 @@
+  ],
+  "contacts": [
+    {
+      "type": "Chair",
+      "name": "Muhammad Q. Zaman",
+      "email": "mzaman@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Jonathan Gribetz",
+      "email": "gribetz@princeton.edu"
+    }
+  ],
+  "req_list": [
+    {
+      "name": "Prerequisites",
+      "max_counted": 1,
+      "min_needed": "ALL",
+      "explanation": "A student who has completed at least one course in the department is eligible to concentrate in Near Eastern studies. This course may be a language class or a course or seminar offered in any of the disciplines covered by the department.",
+      "double_counting_allowed": true,
+      "course_list": [
+        "NES *"
+      ]
+    },
+    {
+      "name": "Language",
+      "max_counted": 1,
+      "min_needed": 1,
+      "explanation": "Students must complete or pass out of four terms (i.e., through the 107 level) of Arabic, Hebrew, Persian, or Turkish. Students are encouraged, albeit not required, to continue language study at the advanced level and to use their chosen Near Eastern language for senior thesis research.",
+      "course_list": [
+        "ARA 107",
+        "HEB 107",
+        "PER 107",
+        "TUR 107"
+      ]
+    },
+    {
+      "name": "Departmentals",
+      "max_counted": 1,
+      "min_needed": "ALL",
+      "explanation": "NES majors take eight departmental courses in Near Eastern Studies.\nA single course may satisfy multiple distribution requirements across these domains. For example, a course focusing on 20th century Turkish literature may count as modern (Historical Periods), Ottoman Empire/Turkey (Regions), and literature (Disciplines).\nA single course may not, however, satisfy more than one distribution requirement within a single domain; a course on Arabic literature that spans the pre-modern and modern periods would thus not satisfy both the pre-modern and modern Historical Periods requirements.\nAt least five departmentals must be NES-prefix or language courses. Up to three may be cognate courses (courses in other departments not cross-listed with NES that focus on the Near East), on the approval of the Department Representative.\nLanguage courses beyond the 107 level count as departmentals, as does elementary and intermediate study of a second Near Eastern language.",
+      "req_list": [
+        {
+          "name": "Seminar in Research Methods",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": "All students are required to take NES 300 (Seminar in Research Methods) in the junior year unless they are studying abroad that semester, in which case they are required to take NES 300 in the senior year.",
+          "course_list": [
+            "NES 300"
+          ]
+        },
+        {
+          "name": "Distribution Requirements",
+          "max_counted": null,
+          "min_needed": null,

```

Is this supposed to be 1 and "ALL"?
Otherwise the counters from its children are passed as is to its parent, which I don't think is what you intended.
```suggestion
          "max_counted": 1,
          "min_needed": "ALL",
```